rtbkit/core/router/router.cc

```diff
@@ -118,7 +125,8 @@ Router(ServiceBase & parent,
        Amount maxBidAmount,
        int secondsUntilSlowMode,
        Amount slowModeAuthorizedMoneyLimit,
-       Seconds augmentationWindow)
+       Seconds augmentationWindow,
+       std::string enableJsonFiltersFile)
```

Comment:
Rather than passing the filters file as a `std::string` to the router's constructor, I think that I would directly pass the `Json::Value` so that the parsing is done by the router_runner and not the router. 

If the value is null, initialize the pool with the default filters.